Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```
@@ -22,207 +20,55 @@ span.bocDateValue
   min-width: 8em;
 }
 
-span.bocDateTimeValue.readOnly,
-span.bocDateValue.readOnly
-{
-  height: inherit;
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
-}
-
-span.bocDateTimeValue.disabled,
-span.bocDateValue.disabled
-{
-}
-
-.mozilla span.bocDateInputWrapper
-{
-  right: 2px;
-}
-
-span.bocDateInputWrapper
-{
-  position: absolute;
-  left: 0;
-  min-width: 4em;
-  width: auto;
-}
-
-span.bocTimeInputWrapper
-{
-  position: absolute;
-  right: 0;
-}
-
-span.bocDateTimeValue span.bocDate24HoursWithSeconds,
-span.bocDateTimeValue span.bocDate12HoursWithSeconds
-{
-  right: 9.0em;
-}
-
-.mozilla span.bocDateTimeValue span.bocDate24HoursWithSeconds,
-.mozilla span.bocDateTimeValue span.bocDate12HoursWithSeconds
-{
-  right: calc(9.0em + 2px);
-}
-
-span.bocDateTimeValue span.bocDate24Hours,
-span.bocDateTimeValue span.bocDate12Hours
-{
-  right: 7.4em;
-}
-
-.mozilla span.bocDateTimeValue span.bocDate24Hours,
-.mozilla span.bocDateTimeValue span.bocDate12Hours
-{
-  right: calc(7.4em + 4px);
-}
-
-span.bocDateValue span.bocDate24Hours,
-span.bocDateValue span.bocDate12Hours
-{
-  right: 2.2em;
-}
-
-.mozilla span.bocDateValue span.bocDate24Hours,
-.mozilla span.bocDateValue span.bocDate12Hours
+a.bocPicker24HoursWithSeconds,
+a.bocPicker12HoursWithSeconds,
+a.bocPicker24Hours,
+a.bocPicker12Hours
 {
-  right: calc(2.2em + 2px);
+  display: flex;
+  align-items: center;
 }
 
-span.bocDateInputWrapper input
+span.bocDateInputWrapper input,
+span.bocTimeInputWrapper input
 {
+  box-sizing: border-box;
   width: 100%;
 }
 
```

```suggestion

span.bocTimeInputWrapper.bocTime24HoursWithSeconds input,
span.bocTimeInputWrapper.bocTime24Hours input
{
  width: calc(4.3em + 2 * var(--remotion-themed-padding));
}

span.bocTimeInputWrapper.bocTime12HoursWithSeconds input,
span.bocTimeInputWrapper.bocTime12Hours input
{
  width: calc(6em + 2 * var(--remotion-themed-padding));
}
```
Actually, I think this would be the best way forward. It doesn't make sense if we have different widths for seconds/no-seconds since they make the layout unstable.